apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -60,30 +65,33 @@ const CurriculumCatalog = ({
                 !!curriculum.grade_levels && !!curriculum.course_version_path
             )
             .map(
-              ({
-                key,
-                image,
-                display_name,
-                display_name_with_latest_year,
-                grade_levels,
-                duration,
-                school_subject,
-                cs_topic,
-                course_version_path,
-                course_version_id,
-                course_id,
-                course_offering_id,
-                script_id,
-                is_standalone_unit,
-                is_translated,
-                //Expanded Card Props
-                device_compatibility,
-                description,
-                professional_learning_program,
-                video,
-                published_date,
-                self_paced_pl_course_offering_path,
-              }) => (
+              (
+                {
+                  key,
+                  image,
+                  display_name,
+                  display_name_with_latest_year,
+                  grade_levels,
+                  duration,
+                  school_subject,
+                  cs_topic,
+                  course_version_path,
+                  course_version_id,
+                  course_id,
+                  course_offering_id,
+                  script_id,
+                  is_standalone_unit,
+                  is_translated,
+                  //Expanded Card Props
+                  device_compatibility,
+                  description,
+                  professional_learning_program,
+                  video,
+                  published_date,
+                  self_paced_pl_course_offering_path,
+                },
+                index
```

Comment:
Could the `handleExpandedCardChange` function work with the course's `key` rather than a new `index`? Then you wouldn't have to add `index` here and everything could rely on the `key` instead?

Reply:
Nice catch, ill make this change!